addons/mass_mailing/static/src/scss/mass_mailing.ui.scss

```diff
@@ -199,6 +199,10 @@ body.editor_enable.o_basic_theme.o_in_iframe {
 
 .oe_structure {
     width: 100%;
+    @media (max-width: 1024px) {
```

Comment:
Xmo says: you should use the break machin bidule from bootstrap, ask people from JS. Or CSS. Or ping Qsm.

Tde globally agrees.